js/ui/status/automaticUpdates.js

```diff
@@ -286,12 +387,22 @@ var Indicator = new Lang.Class({
             return AutomaticUpdatesState.IDLE;
     },
 
-    _getActiveConnection: function() {
-        let activeConnection = this._client.get_primary_connection();
+    _isNewConnection: function() {
+        let connection = this._getActiveConnection();
+
+        if (!connection)
+            return false;
 
-        if (!activeConnection)
-            activeConnection = this._client.get_activating_connection();
+        let userSetting = connection.get_setting(NM.SettingUser.$gtype);
+
+        if (!userSetting)
+            return true;
```

Comment:
Shouldn't this return `false`? If you can't get a valid `userSetting` here you can't really tell whether this is a new connection or not, right?

Reply:
The code assumes that, if there is no `NMSettingUser`, it is probably that the user connected to it in the past with Endless OS, __before the automatic updates feature__, and we didn't touch the connection. So, it must be notified.